### ISSUE TITLE
Theme change not working on Firefox and Edge

### ISSUE THREAD

**Issue by bieniu** [andrey-git/home-assistant-custom-ui#133]
I use Custom UI (local install with Customizer component) to change theme of sensors icons. It works perfectly on Chrome/Opera but on Firefox and Edge it’s completely ignored.

### Checklist
- [X] I'm running the latest version of CustomUI ([Update guide](https://github.com/andrey-git/home-assistant-custom-ui/blob/master/docs/installing.md#updating)) or using a specific [release](https://github.com/andrey-git/home-assistant-custom-ui/releases) that is not marked as "Broken".
- [X] I tried to force-refresh (Ctrl+Shift+R / Ctrl+F5) the browser
- [X] (Optional, but recommended) I'm using Chrome or tried to reproduce the feature on Chrome.

**Browser + Version:** Firefox 60.0.1

**CustomUI version:** 20180521

**Home Assistant release (`hass --version`):** 0.69.1

**Problem-relevant `configuration.yaml` entries:**
```
homeassistant:
  customize_glob:
    'binary_sensor.*_status':
      device_class: connectivity
      custom_ui_state_card: state-card-custom-ui
      theme: 'grey'

frontend:
  javascript_version: auto
  themes:
    grey:
      paper-item-icon-color: '#969696'
      paper-item-icon-active-color: '#44739e'

customizer:
  custom_ui: local
```

**Any errors from browser Javascript console:** no errors

**Comment by andrey-git** [comment]
Does "regular" (i.e not per-entity) theming work for you in FF? i.e. if you select this "grey" theme - do all icons become grey?

I tried in FF52 on Linux with HA 0.70 and everything works fine.

**Comment by bieniu** [comment]
Yes, when I change theme in frontend into 'gray' the icons of all entities with state 'off' change color to gray. I tried in FF60, FF52 and Edge and always the same issue.

**Comment by andrey-git** [comment]
Could you attach a screenshot from Chrome where it is working, so I better understand the setting?

**Comment by bieniu** [comment]
Chrome - icons of entities with state 'off' are grey, with state 'on' - blue
![chrome](https://user-images.githubusercontent.com/478555/40541257-5aafc5f2-601b-11e8-8df3-f38d8d07db15.PNG)

Firefox - icons of entities with state 'off' are blue, with state 'on' - yellow
![firefox](https://user-images.githubusercontent.com/478555/40541376-d1a6f388-601b-11e8-9ef9-94fff62fab30.PNG)

**Comment by andrey-git** [comment]
Thanks, I reproduced the issue, trying to debug now...

**Comment by andrey-git** [comment]
Fixed in release 20170528
